lib/puppet/face/strings.rb

```diff
@@ -50,6 +50,7 @@ def check_required_features
       # For now, assume the remaining positional args are a list of manifest
       # and ruby files to parse.
       yard_args = (args.empty? ? MODULE_SOURCEFILES : args)
+      class YARD::Logger; def progress(*args); end; end
```

Comment:
ah ok - then it is better to have it as a one-liner :-) if useful later - comment it out and have a comment # For debugging, uncomment to .... or somesuch? I have used that in the parser in a place where it was difficult to remember where to add it back in when doing certain kinds of debugging. - just a though.